### PR TITLE
proposal:configuration which allow to run and debug unit tests from VSCode

### PR DESCRIPTION
**Pull request by avisiedo**
- Set configuration for .gitignore
- For tests, set CONFIG_PATH to point out the current 'configs` direcotory
  using the absolute workspace path.

### CODE REVIEW

**Comment by jlsherrill** [comment]
@Andrewgdewar mind looking at this since you use vscode?

**Comment by Andrewgdewar** [comment]
> @Andrewgdewar mind looking at this since you use vscode?

Will do.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "go.testEnvVars": {
+        "CONFIG_PATH": "${workspaceFolder}/configs"
```
Comment (Andrewgdewar) [inline .vscode/settings.json:3]: Would "./configs" not work here?

For the .vscode folder to work I thought you had to have the project folder open anyway.

Comment (avisiedo) [reply .vscode/settings.json:3]: Sorry for the delay mate! I was getting an error when launching the tests in visual studio code; by the situation that I got and the observations, from the ide the test binaries are generated in each package directory; when the tests are run from the ide, the current working directory is changed to the package directory, making the relative reference wrong.

It happens when we try to debug a test from the test lab view. I attach a screenshot

Comment (avisiedo) [reply .vscode/settings.json:3]: image

**Review by Andrewgdewar — APPROVED** [review]
Tested, LGTM

**Comment by avisiedo** [comment]
@Andrewgdewar thanks for the review!

**Pull request merged**